http/label_service.go

```diff
@@ -620,9 +620,8 @@ func (s *LabelService) CreateLabelMapping(ctx context.Context, m *influxdb.Label
 		return err
 	}
 
-	urlPath := resourceIDPath(m.ResourceType, m.ResourceID, "labels")
 	return s.Client.
-		PostJSON(m, urlPath).
+		PostJSON(m, string(m.ResourceType), m.ResourceID.String(), "labels").
```

Comment:
these were hitting wrong routes. The tests in pkger failed when trying to validate label mappings with the http client as the label service dependency. These are fixed and tests are now passing. There is a big gap in tests for this inside `http` pkg itself though. I did not go back and add that at this point.